config/locales/two_factor_authentication/en.yml

```diff
@@ -68,8 +68,8 @@ en:
     phone_voice_label: Phone number
     piv_cac_fallback:
       link: Use your PIV/CAC instead
-      text_html: Do you have your PIV/CAC? %{link}
       question: Don't have your piv/cac card available?
+      text_html: Do you have your PIV/CAC? %{link}
```

Comment:
Was this reordering done manually or is there a bug in `make normalize_yaml`? It should be sorting keys alphabetically. Same question for the 2 files below.